### PR TITLE
Styled components batch4.1

### PR DESCRIPTION
**Pull request by kwasniew**

### CODE REVIEW

File: frontend/src/component/menu/Header/Header.tsx [hunk 1]
```diff
@@ -158,9 +158,7 @@ const Header: VFC = () => {
                 <StyledContainer>
                     <Tooltip title="Menu" arrow>
                         <IconButton
-                            sx={theme => ({
-                                color: theme.palette.text.tertiaryContrast,
-                            })}
+                            sx={{ color: 'black' }}
```
Comment (kwasniew) [inline frontend/src/component/menu/Header/Header.tsx:161]: I broke it previously

File: frontend/src/component/context/ContectFormChip/ContextFormChip.tsx [hunk 2]
```diff
@@ -1,34 +1,69 @@
...
+}));
+
+const StyledDescription = styled('div')(({ theme }) => ({
+    fontSize: theme.fontSizes.smallerBody,
+    color: theme.palette.neutral.main,
+}));
+
+const StyledButton = styled('button')(({ theme }) => ({
+    all: 'unset',
+    lineHeight: 0.1,
```
Comment (Tymek) [inline frontend/src/component/context/ContectFormChip/ContextFormChip.tsx:35]: This value looks like a hack

File: frontend/src/component/context/ContextForm/ContextForm.tsx [hunk 3]
```diff
@@ -27,6 +33,52 @@ interface IContextForm {
...
+
+const StyledTagContainer = styled('div')(({ theme }) => ({
+    display: 'grid',
+    gridTemplateColumns: '1fr auto',
+    gap: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+}));
+
+const StyledInputHeader = styled('p')({
+    marginBottom: '0.3rem',
```
Comment (Tymek) [inline frontend/src/component/context/ContextForm/ContextForm.tsx:63]: Maybe this element can be aligned in a different way. `4.8px` looks off.

File: frontend/src/component/feature/FeatureForm/FeatureForm.tsx [hunk 4]
```diff
@@ -39,6 +40,66 @@ interface IFeatureToggleForm {
...
+
+const StyledFormControl = styled(FormControl)(({ theme }) => ({
+    width: '100%',
+    marginBottom: theme.spacing(2),
+}));
+
+const styledSelectInput = (theme: Theme) => ({
+    marginBottom: theme.spacing(2),
+    minWidth: '400px',
+    [theme.breakpoints.down(600)]: {
```
Comment (Tymek) [inline frontend/src/component/feature/FeatureForm/FeatureForm.tsx:71]:
```suggestion
    [theme.breakpoints.down('sm')]: {
```

File: frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx [hunk 5]
```diff
@@ -23,12 +22,33 @@ interface IFeatureStrategyEmptyProps {
...
+const StyledContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingTop: theme.spacing(2),
+}));
+
+const StyledTitle = styled('div')(({ theme }) => ({
+    fontSize: theme.fontSizes.bodySize,
```
Comment (Tymek) [inline frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx:34]:
```suggestion
    fontSize: theme.typography.body1.fontSize,
```

File: frontend/src/component/feature/FeatureStrategy/FeatureStrategySegment/FeatureStrategySegment.tsx [hunk 6]
```diff
@@ -6,22 +6,24 @@ import {
...
-import { Divider, Typography } from '@mui/material';
+import { Divider, styled, Typography } from '@mui/material';
 
 interface IFeatureStrategySegmentProps {
     segments: ISegment[];
     setSegments: React.Dispatch<React.SetStateAction<ISegment[]>>;
 }
 
+const StyledDivider = styled(Divider)(({ theme }) => ({
+    fontSize: theme.fontSizes.smallBody,
```
Comment (Tymek) [inline frontend/src/component/feature/FeatureStrategy/FeatureStrategySegment/FeatureStrategySegment.tsx:19]:
```suggestion
    fontSize: theme.typography.body2.fontSize,
```

File: frontend/src/component/menu/Header/Header.tsx [hunk 7]
```diff
@@ -158,9 +156,7 @@ const Header: VFC = () => {
...
```
Comment (Tymek) [inline frontend/src/component/menu/Header/Header.tsx:159]:
```suggestion
                            sx={{ color: theme => theme.palette.text.primary }}
```
or `theme.palette.common.black`

**Review by Tymek — APPROVED** [review]

File: frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsChips/FeatureMetricsChips.tsx [hunk 8]
```diff
@@ -10,15 +9,36 @@ interface IFeatureMetricsChipsProps {
     setValue: (value: string) => void;
 }
 
+const StyledTitle = styled('h2')(({ theme }) => ({
+    margin: 0,
+    marginBottom: theme.spacing(1),
+    fontSize: theme.fontSizes.smallBody,
```
[the remark this answers is not in the document]
Comment (kwasniew) [reply frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsChips/FeatureMetricsChips.tsx:15]: @Tymek what is body2.fontSize?

File: frontend/src/component/context/ContectFormChip/ContextFormChip.tsx (hunk 2 above)
Comment (kwasniew) [reply frontend/src/component/context/ContectFormChip/ContextFormChip.tsx:35]: I know. However in this PR I didn't intend to resolve all the hacks. It was mostly styled components and replacing color/spacing that had obvious values.

File: frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx (hunk 5 above)
Comment (kwasniew) [reply frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx:34]: we don't have body1.fontSize in the entire codebase. Not sure what that is

**Pull request merged**